utils/handy.ts

```diff
@@ -3,11 +3,12 @@ import axios from "axios";
 
 /**
  * Retrieve the IP address of the current machine
- * @returns {Promise<string> as IP address}
+ * @returns {Promise<string>} as IP address
  */
 export async function IPify(): Promise<string> {
   try {
-    return await (await axios.get("https://api.ipify.org?format=json")).data.ip;
+    const response = await axios.get("https://api.ipify.org?format=json");
```

Comment:
Don't stick to code that has not been referenced in the issue ( #1 )

Reply:
Okay!